users/urls.py

```diff
@@ -10,4 +11,9 @@
         name="retrieve-update-destroy",
     ),
     path("", UserAPIView.as_view(), name="profile-register"),
+    path(
+        "auth/activate/<str:uidb64>/<str:token>/",
```

Comment:
Is `str` okay or maybe it's better to use regex like `(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$`?